use/toolboxBlocks.md

```diff
@@ -0,0 +1,26 @@
+# Logic Blocks
+
+Reference documentation about the functionality of each block in system.
+
+|   |   |   |
```

Comment:
Missing a table header if you want one, if not I think this line could be deleted

Reply:
(Right now it's just an empty row)

Reply:
unfortunately when I remove that row it doesn't render as a table in the environments I've tried - apparently only some parsers support tables without headers

Reply:
I think it would be nice to have a header of Name Icon Description then since it just looks like an error right now